### PR TITLE
Lobby

### PR DESCRIPTION
**Pull request by bpetetot**
#42 finalize lobby screen

### CODE REVIEW

**Comment by bpetetot** [comment]
@fabienjuif @frinyvonnick what do you think about this lobby architecture and gui components ?

File: package.json [hunk 1]
```diff
@@ -92,17 +92,16 @@
...
-      ],
+      "semi": [ 2, "never" ],
       "arrow-body-style": 0,
       "import/no-named-as-default": 0,
       "import/prefer-default-export": 0,
       "import/no-unresolved": 0,
       "generator-star-spacing": 0,
       "babel/generator-star-spacing": 1,
       "react/require-extension": 0,
+      "no-underscore-dangle": 0,
```
Comment (fabienjuif) [inline package.json]: I prefer to let this activated

File: src/gui/components/Button.js [hunk 2]
```diff
@@ -0,0 +1,50 @@
+import { Text, Graphics } from 'pixi.js'
+import Component from './Component'
+
+export const DEFAULT_OPTIONS = {
```
Comment (fabienjuif) [inline src/gui/components/Button.js]: Why do you export `DEFAULT_OPTIONS` in all components ?
How do you think about a static property (like react do) on the class ?

**Review by fabienjuif — CHANGES_REQUESTED** [review]
First review

**Comment by fabienjuif** [comment]
Seems pretty cool overall :)

**Comment by bpetetot** [comment]
This PR starts to be very big, can you do a review to merge it... then I will make smaller PR

**Review by fabienjuif — APPROVED** [review]
When I see all the code to do a simple UI I ask myself if this is not worth it to have an UI lib eventually.

**Comment by bpetetot** [comment]
I agree but I searched for UI libs with pixi.js and nothing seems stable.....

File: src/gui/renderer/entities/walls.js [hunk 3]
```diff
@@ -1,5 +1,5 @@
 import { Graphics } from 'pixi.js'
-import { convert } from '../convertor'
+import { convert } from '../../../utils'
```
Comment (frinyvonnick) [inline src/gui/renderer/entities/walls.js:2]: Could we configure webpack to import from root ?

**Review by frinyvonnick — APPROVED** [review]

File: src/gui/renderer/entities/walls.js (hunk 3 above)
Comment (fabienjuif) [reply src/gui/renderer/entities/walls.js:2]: The issue is on you :p https://github.com/bpetetot/pongmmo/issues/21

Comment (bpetetot) [reply src/gui/renderer/entities/walls.js:2]: Already identified : #21

**Pull request merged**